src/CommonITILTask.php

```diff
@@ -266,6 +266,29 @@ public function post_deleteFromDB()
         }
     }
 
+    /**
+     * Handle the task duration and planned duration logic.
+     *
+     * This function ensures a bi-directional link between the task duration and the planned duration.
+     * These two fields can be a bit redundant when task planning is enabled.
+     *
+     * @param array $input The input array, passed by reference.
+     * @param int $timestart The start time of the task.
+     * @param int $timeend The end time of the task.
+     * @return void
+     */
+    public function handleTaskDuration(&$input, $timestart, $timeend)
```

Comment:
If method is used only privately, please declare it as private (it ensure that we can change its signature without affecting plugins).

There is a `$this->callPrivateMethod($instance, $methodName, ...$args)` method that can be used in test suite to test private methods.

```suggestion
    private function handleTaskDuration(array &$input, int $timestart, int $timeend): void
```